### DISCUSSION TITLE
npx react-native doctor issues fixed but emulator (virtual device) refuses to launch

### DISCUSSION THREAD

**Discussion by JulianHSC** [microsoft/react-native-windows#12917] [Q&A] [answered]
Hello Chris or team,

I persistently get this problem. I've learned how to kill a process in Windows but i can't start my emulator. What's happening? Don't pay attention to the build being successful. Pay attention to the fact that emulator just can't start. Why? I have removed the messages below the build being successful as it's not important. (Of course anyone else with the same problem can easily add it.)

C:\Z-Android\RN_TextI>npx react-native run-android
The filename, directory name, or volume label syntax is incorrect.
info Launching emulator...
error Failed to launch emulator. Reason: No emulators found as an output of `emulator -list-avds`.
warn Please launch an emulator manually or connect a device. Otherwise app may fail to launch.
info Installing the app...

> Task :app:installDebug
Installing APK 'app-debug.apk' on 'Android_11_Pixel_6a_API_30(AVD) - 11' for :app:debug
Installed on 1 device.

BUILD SUCCESSFUL in 8s

**Comment by JulianHSC** [comment]
I managed to solve my own problems but just be warned that many solutions on the internet are suggestions. There's no guarantee they will work. Solved.

**Comment by JulianHSC** [comment]
Reopen to close case.

**Comment by chrisglein** [comment, marked as answer]
This is the wrong repo for support for Android development on Windows. Closing discussion.
